tests/util/test_yaml.py

```diff
@@ -372,6 +386,17 @@ def test_secrets_keyring(self):
         _yaml = load_yaml(self._yaml_path, yaml_str)
         self.assertEqual({'http': {'api_password': 'yeah'}}, _yaml)
 
+    def test_secrets_credstash(self):
```

Comment:
It uses the `self._yaml_path` from setup, which itself uses some other parts of the setup.

The path is needed to ensure fall-through when a secret isn't in the local file, that `credstash.getSecret` is called